syne_tune/optimizer/schedulers/multiobjective/utils.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import numpy as np
+
+from syne_tune.try_import import try_import_moo_message
+
+try:
+    from pymoo.indicators.hv import HV
+except ImportError:
+    print(try_import_moo_message())
+
+
+EPSILON = 1e-6
+
+
+def hypervolume(
```

Comment:
Can you expand on what this function is doing? Most of the heavy-lifting is coming from pymoo's HV function, but what does it mean when reference_point is None?

Reply:
It is computing hypervolume indicator for the provided point and reference. Its really just a convinienece wrapper around pymoo. The thinking is that if we want to replace the library for something else like pygmo we can just replace the implementation here rather than scount for it in the code. 

Re second question updated the docstring to be clearer